src/lib/utils/meta_tables/meta_settings_table.cpp

```diff
@@ -8,7 +8,8 @@ namespace opossum {
 MetaSettingsTable::MetaSettingsTable()
     : AbstractMetaTable(TableColumnDefinitions{{"name", DataType::String, false},
                                                {"value", DataType::String, false},
-                                               {"description", DataType::String, false}}) {}
+                                               {"description", DataType::String, false},
+                                               {"display name", DataType::String, false}}) {}
```

Comment:
Please don't use spaces in column names.

Reply:
Can you help me to understand this. Do we insert a new row to settings every time an update is triggered in the frontend? Or is this the place where we register settings?

Reply:
No, meta tables do not follow the insert-only approach. Instead, updates are handled by the addressed table's `_on_update()` method. In this case the value of the Setting in the SettingsManager is changed.